### DISCUSSION TITLE
option to set order for forum itself

### DISCUSSION THREAD

**Discussion by Jimmi08** [e107inc/e107#4674] [Ideas]
I have 2 forums in maybe 20 other forums when I need to get topics sorted by alphabet.  

Could be "order by" option added to forum itself?  

I have no idea how to solve this.  For now.

**Comment by Moc** [comment]
Sounds like a useful feature. Is this this forums (categories) or forum topics?

**Reply by Jimmi08** [reply]
Not clear what you are asking.  I needed to set an order for the forum and display topics in this forum according to this. 

Maybe this helps:  
I copied
`function forumGetThreads()`
to 
`function forumGetThreadsByAlphabet() -  with changed order by`
and then in forum_viewforum.php I changed

```
if($forumId == 70) {
    	$threadList = $forum->forumGetThreadsByAlphabet($forumId, $threadFrom, $view, $threadFilter);    
    }
    else {
    	$threadList = $forum->forumGetThreads($forumId, $threadFrom, $view, $threadFilter);
    }
```

It was all.
